examples/persistence-service-catalog.md

```diff
@@ -8,19 +8,20 @@
 
 - odo which will provide the ability to run the spring boot application on Openshift and also create the PostgreSQL database
 
+To build odo from source, perform the following commands (requires golang tools to be setup properly)
+
 ```bash
 cd $GOPATH/src/github.com/redhat-developer
 git clone https://github.com/redhat-developer/odo.git && cd odo
 make install && sudo cp $GOPATH/bin/odo /usr/local/bin
 ```
 
-- Create a Minishift's vm running okd 3.10 and service catalog 
+Alternatively you can install the latest odo release (requires you have [jq](https://stedolan.github.io/jq/) setup)
 
 ```bash
-minishift addons enable xpaas
-minishift addons enable admin-user
-MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags="--enable=service-catalog,automation-service-broker" 
```

Comment:
Why have you removed the minishift's instructions ?

Reply:
Mea cupla! I will bring them back :)

Reply:
There is an issue to install with minishift the service catalog using this command 

```
MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags="--enable=service-catalog,automation-service-broker"
...
Error during 'cluster up' execution: Error starting the cluster. ssh command error:
command : /var/lib/minishift/bin/oc cluster up --base-dir /var/lib/minishift/base --image 'openshift/origin-${component}:v3.10.0' --public-hostname 192.168.65.26 --routing-suffix 192.168.65.26.nip.io --enable=service-catalog,automation-service-broker
err     : exit status 1
output  : error: cannot use --enable when the cluster is already initialized, use cluster add instead
 ```

Reply:
see -> https://docs.okd.io/latest/minishift/command-ref/minishift_openshift_component_add.html

Reply:
Syntax is

```
minishift openshift component add service-catalog
minishift openshift component automation-service-broker
```

Reply:
I believe that we shouldn't complicate the workflow. We could just mention that if there is a previous minishift VM, it needs to be deleted.

Reply:
We can then mention to start the minishift vm and if not yet done to add the components

```
minishift openshift component add service-catalog
minishift openshift component automation-service-broker
```

Reply:
OK, I will add them